app/domain/export/pdf/passes/membership/person.rb

```diff
@@ -11,29 +11,26 @@ class Person < Export::Pdf::Section
     alias person model
 
     def render
```

Comment:
Hier müssen wir uns noch was überlegen bzw mit dem SAC klären. 

Aktuell führt ein längere Name zu overflows (siehe screenshot)  
![image](https://github.com/hitobito/hitobito_sac_cas/assets/180780/6f8da611-42c1-4f30-af9a-1c10b4885a0d)
und wenns noch länger wird dann knallts  `Table's width was set too small to contain its contents (min width 263.638, requested 260)`